### ISSUE TITLE
Comparison with other libs

### ISSUE THREAD

**Issue by webknjaz** [DigitalBrainJS/c-promise#1]
Hey, it'd be great to see how this library compares with other promise cancellation implementations. In particular, I'm interested in the difference with http://bluebirdjs.com/docs/api/cancellation.html.
I think it could be useful to have some sort of a comparison table in the README.

**Comment by DigitalBrainJS** [comment]
@webknjaz  Yes, for sure. I've thought about that, but it's not easy to find an expressive/concise way to do this without scaring off potential users with unnecessary details. Readme already looks too complicated, so seems I should make the comparison table and quickstart section and move the rest into separate .md documents.

**Comment by DigitalBrainJS** [comment]
@webknjaz Done. Check it out https://github.com/DigitalBrainJS/c-promise/tree/master#comparison-table

**Comment by khamoshimasih-hub** [comment]
Tanks for progrm
